gfx/drivers/ps2_gfx.c

```diff
@@ -89,8 +90,7 @@ static void deinitTexture(GSTEXTURE *texture) {
    texture->Clut = NULL;
 }
 
-static void color_correction32(uint32_t *buffer, uint32_t dimensions)
-{
+static void color_correction32(uint32_t *buffer, uint32_t dimensions) {
```

Comment:
I don't really mind either way, but this makes this file inconsistent with most of the code base. I have a feeling it could get changed back in the future without realizing it was intentional.

Reply:
Sorry I will create today a PR fixing it.
You know, this is just bad habits. I'm working in other projects where a lot of code style guidelines are different.

Reply:
Fully understandable and not a big deal either way. :)